client/src/main/java/com/microsoft/durabletask/DurableTaskGrpcWorker.java

```diff
@@ -214,6 +214,8 @@ private void closeSideCarChannel() {
     }
 
     private void shutDownWorkerPool() {
+        logger.log(Level.WARNING, "ExecutorService shutdown initiated. No new tasks will be accepted");
```

Comment:
This is not correct. Now this warning will be logged every time a user shuts down their worker, but we only want to log the warning when the worker pool was shut down *unexpectedly*.

Reply:
I opened a [PR here](https://github.com/dapr/durabletask-java/pull/4). I think this addresses your comment.